src/main/java/com/iota/iri/storage/Persistable.java

```diff
@@ -47,4 +48,19 @@
      * false if we should replace the old value
      */
     boolean merge();
+
+    /**
+     * Merges source object into this object.
+     * @param source
+     * @return
```

Comment:
the updated persistable, which contains both. Or {@code null} if the persistable cannot be merged